app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,46 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end    
+
+  def show
+    @question = Question.find(params[:id])
+    
+    if @question.resolved 
```

Comment:
You are correct that it's a code smell to have significant Ruby in the view.  Most code and business logic should be in the model or controller, however the view layer should be responsible for displaying what the user sees, so some ruby code may be required.  In this example `@question.resolved` is a business logic decision that indicates the state of a question, so that should be decided in the model (which it is).  However choosing the text that should be displayed to a user based on whether the question was resloved or not seems like part of the view.  

These decision are definitely subjective, and you could easily get other answers depending on who you talk to, but its something you will learn over time.

I'm not following what you were saying about merging into the wrong branch.  Did you make the change? If so could you link me to the appropriate commit?  Feel free to reach out if you get stuck in a weird git state and aren't sure how to resolve.